data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3447 @@
+
+
+
+mission "Unfettered: Attracted Attention"
+	invisible
+	landing
+	source "Freedom"
+	destination "Darkcloak"
+	to offer
+		has "Unfettered: Prove your strength 2: active"
+	on offer
+		require "Jump Drive" 1
+	on enter "Almaaz"
+		dialog `The Unfettered comes to the bridge to warn you, "There are even more unfettered humans than anticipated. I told you not to come here with a jump drive!"`
+	npc evade
+		government "Pirate"
+		system "Almaaz"
+		personality disables plunders vindictive
+		fleet "Large Northern Pirates"
+
+mission "Unfettered: Jump Drive Retrieval"
+	invisible
+	landing
+	source "Freedom"
+	destination "Darkcloak"
+	to offer
+		has "Unfettered: Prove Your Strength 2: active"
+	on offer
+		conversation
+			`Following the advice of the Hai, you park your ship close to the hidden depot in case something goes wrong, and then start following him to the hatch. He is moving at a quick pace, you have difficulties following him, even given the fact he is wearing a combat suit.`
+			choice
+				`	"Who made this vault, what should I expect?"`
+				`	(No time for talking.)`
+					goto outside
+			`	"That is an information I cannot trust anyone with, not even you, and especially not here. I do not want the owner of this cache to be in danger whilst we need jump drives to expand. Assume it is owned by unfettered humans, or pirates as I think you call them, and that we are ready to face them," he tells you. He is fully armoured, meaning as long as you stay with him, everything should be fine.`
+			
+			label outside
+			`	Before exiting, he warns you, "Whatever happens, do not try anything stupid, and stay behind me at all times." He seems worried and keeps looking around, without making a sound. Well as little sound as possible, anyway, given he is in quite an impressive suit. It must have been crafted specifically for him in order to be so discrete. He does not have his helmet on, probably in order to see his surroundings better.`
+			choice
+				`	"Is everything alright?"`
+				`	"There is no need to be worried, that cache is well hidden."`
+				`	(Keep walking, ready to reach for my gun.)`
+					goto discrete
+			`	You are not sure what kind of sign he just did, but it seems pretty clear he wants you to keep quiet. Looking around, you notice the two of you have attracted quite a bit of attention from the locals. While they all seem to have nothing to lose, the vast majority of them are very young and probably not the hardened criminals you had expected.`
```

Comment:
There are locals? Where did they come from? Wasn't this a hidden depot?

Reply:
Yeah but for discretion you don't land on top of it (at least not until you are sure it is safe; to get the JD, as whoever makes the law on this pirate world is not aware of it).

This is on the way there; you go discretely first to check if it isn't a trap (which it is)

Reply:
Yeah ok none of that was clear - it read like you'd landed somewhere off-grid and unobserved, explored stealthily around, and suddenly there's crowds of locals out of nowhere.
Needs to be much clearer what's going on from the start.

Reply:
It probably wasn't clear in my head when I wrote it, good catch
It is probable you were followed so landing off grid is pretty much saying "plz, come and take my ship"